### ISSUE TITLE
Computed attributes ignore `field` property during update

### ISSUE THREAD

**Issue by sam3d** [tywalch/electrodb#281]
**Describe the bug**
I'd expect an attribute to adhere to the `field` property at all times - whether or not it's a computed attribute. However, computed attributes seem to ignore the `field` property during an update operation when their update is triggered from another attribute being set.

**ElectroDB Version**
2.8.0

**ElectroDB Playground Link**

[https://electrodb.fun/?#code/JYWwDg9gTgLgB...](https://electrodb.fun/?#code/JYWwDg9gTgLgBAbzgUQHY2DAnnAvnAMyghDgCIBTAGwoGMZiATAIzIG4AoD2iVAZ3gwAhnwDWfOAF44qCgHcU6TFgAUHOInUa4ICI2oAuTdu0Ul2I2WFi+ZADRaTANwpQ+wXpYCM9x9r6uTsC0FJbWokJgYL4muA4mQjAMwMwArjAUfEYIfhrAjNlw2GCh5AJQwKgA5mR48SZwFAAeYMBQmQCCMIXFpWTllTV2cPoEQqlU3XAqAJRSAHwy8nAAIokUswB0MBAAkgDKAPL7ydWzdblwVBC0QlQAYsDUjAByQiClOQ3avZaoqSBmK4Yt9CE8qAVyAAVKEAGXscAA9Ii4FCABbACR8NEQCaMOBAuCpAL4uRosxEsCMRKDS4aOSJWhoowAbUoLTanRgZAAuvVvgEpioAPrDJDNVrtPhdPBzSSLNYZTZgIRuDYSznSmAzfnaOJ+fUmSr6JqZbKXZhYXaQr6gsCiQoEcGQsgABQA0gieOAIO4MqyyPleRdQXAxI7nZZ9p7ht7IH7SiyeSGGoa9fU00hhMwaGERBEorV9TNOBxwnxNrR2usVEh8mFMtzZZsqhBZqXkXAAKpUmnVOAAAw17UYXQHRQqVSqrgkQlQlOpGTgEAIg+utwezreHwHwzJwTRcEwASoBA4nexuIhBIoYaELnxIkHMNh4+YIgo+N4RXJg6dz3HRJkjSDJhhAo8YAAcixe9PzgERzxRAd1zuR5nm3Chxx2EYsFQd4IDLfMK1SXsMlrI8XQyAQixmTZBXI4cuSMWQFEVDZaJ2A5jlOKpzlwWjW3bDggA)

**Entity/Service Definitions**

```ts
import { Entity } from "electrodb";

const tasks = new Entity(
  {
    model: {
      entity: "tasks",
      version: "1",
      service: "taskapp",
    },
    attributes: {
      id: { type: "string" },
      expiresAt: { type: "string", default: () => new Date().toISOString() },
      localFieldName: {
        type: "number",
        field: "TTL", // This should be used when updating
        watch: ["expiresAt"],
        set: (_, { expiresAt }) => Date.parse(expiresAt),
      },
    },
    indexes: {
      byId: {
        pk: { field: "PK", composite: ["id"] },
        sk: { field: "SK", composite: [] },
      },
    },
  },
  { table: "taskapp" },
);

tasks.create({ id: "test" }).go();

// Updating `expiredAt` triggers an update of `localFieldName`, which itself
// should be saved as `TTL` based on the `field` attribute, but it's saved as
// `localFieldName` to dynamodb
tasks.update({ id: "test" }).set({ expiresAt: new Date().toISOString() }).go();
```

**Comment by tywalch** [comment]
Thank you! I will make this a priority!

**Comment by tywalch** [comment]
This a great find 👍

**Comment by tywalch** [comment]
I have addressed this issue in version `2.8.2`, let me know if that fixes your issue 👍

**Comment by sam3d** [comment]
Wow that was _so fast_ 😱 That works perfectly for me now, thank you so much !!
